app/views/navbar.scala.html

```diff
@@ -181,41 +181,41 @@
 
     // Triggered when 'Sign In' in navbar is clicked
     // Logs "Click_module=SignIn_location=Navbar_route=</|/audit|/help|...>"
-    $("#sign-in-button").on('click', function(){
```

Comment:
These are actually necessary. They log into the `webpage_activity` table when someone clicks on the various links. Is there a reason why you didn't think they were necessary?

Reply:
My bad, for some reasons I thought it said `loadWebpageActivity` instead of `logWebpageActivity` so I was confused as I thought clicking on the links itself redirected to the page already.